app/components/vote-overview.ts

```diff
@@ -19,6 +20,47 @@ export default class VoteOverview extends Component<ArgsInterface> {
     return this.args.numberOfProponents || 0;
   }
 
+  get numberOfAbstentionsGraphValue() {
+    const totalValue =
+      (this.args.numberOfProponents || 0) +
+      (this.args.numberOfOpponents || 0) +
+      (this.args.numberOfAbstentions || 0);
+    const abstentionsValue = this.args.numberOfAbstentions || 0;
+
+    return (abstentionsValue / totalValue) * 158;
+  }
+
+  get numberOfOpponentsGraphValue() {
+    const totalValue =
+      (this.args.numberOfProponents || 0) +
+      (this.args.numberOfOpponents || 0) +
+      (this.args.numberOfAbstentions || 0);
+    const opponentsValue = this.args.numberOfOpponents || 0;
+
+    return (opponentsValue / totalValue) * 158;
+  }
+
+  get numberOfProponentsGraphValue() {
+    const totalValue =
+      (this.args.numberOfProponents || 0) +
+      (this.args.numberOfOpponents || 0) +
+      (this.args.numberOfAbstentions || 0);
+    const proponentsValue = this.args.numberOfProponents || 0;
+
+    return (proponentsValue / totalValue) * 158;
+  }
+
+  @action
+  bake() {
+    [
+      { key: '--value-abstentions', val: this.numberOfAbstentionsGraphValue },
+      { key: '--value-opponents', val: this.numberOfOpponentsGraphValue },
+      { key: '--value-proponents', val: this.numberOfProponentsGraphValue },
+    ].forEach(({ key, val }) => {
+      document.documentElement.style.setProperty(key, val.toString());
+    });
+  }
```

Comment:
I think this can also be called from the constructor to have the same effect without having to use `{{did-insert}}`.